base/ltsect.dtx

```diff
@@ -815,20 +815,70 @@
 %    \end{macrocode}
 %  \end{macro}
 %
-% \changes{v1.0c}{1994/05/20}{Correct setting of \cs{protect}.}
-% \changes{v1.0h}{1994/11/04}{Added \cs{protected@write} to
-%    \cs{addtocontents}.  ASAJ.}
+%
+%
+%
+%
+%    \begin{macrocode}
+%    \end{macrocode}
```

Comment:
Empty `macrocode` env. Why?

Reply:
leftover from making the rollbacks I guess